src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1623,4 +1648,37 @@ export class PatientViewPageStore {
         },
         []
     );
+
+    // if we use grch37(default), grch38GenomeNexusUrl will be undefined
+    @computed get grch38GenomeNexusUrl() {
```

Comment:
This value is serving as the "source of truth" of the current build.  I don't that's abstract enough.  In other words, I'm a developer, i want to know what the current build is.  I wouldn't think to check whether grch38GenomeNexusUrl is undefined.  I would be looking for "currentReferenceGenemeBuild" or some such abstract property. It can contain the same logic here (based on study result).

Reply:
updated